keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/image/ArtifactVersionDeployed.kt

```diff
@@ -25,6 +25,6 @@ import com.netflix.spinnaker.keel.api.ResourceId
  */
 data class ArtifactVersionDeployed(
   val resourceId: ResourceId,
-  val imageId: String,
+  val artifactVersion: String,
```

Comment:
Since this event needs to work across regions it no longer makes sense to use `imageId`. Also it saves us using CloudDriver to look up the appVersion in the event handler.